### PR TITLE
Bugfix/no support of wildcard directly

### PR DESCRIPTION
**Pull request by MartinCarpentier**
The dotnet nuget push command does not allow wildcards.

I updated the release action with some powershell code, that will function the same way as I expect the wildcard was meant to function.
It uses Get-ChildItem to find the nuget package, and pushes the top 1 result.

Tested on my own fork, and it seems like it finds the correct package, but have not tested the nuget push itself.

### CODE REVIEW

**Comment by MartinCarpentier** [comment]
@matt-lethargic This merge request should fix the release part. (hopefully)

**Review by matt-lethargic — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -48,4 +48,8 @@ jobs:
           }
           dotnet pack -v normal -c Release --include-symbols --include-source -p:PackageVersion=$VERSION -o nupkg src/${{ env.PROJECT_NAME }}/${{ env.PROJECT_NAME }}.csproj
       - name: Push to Nuget
-        run: dotnet nuget push nupkg/*.nupkg --source https://api.nuget.org/v3/index.json --api-key ${{secrets.NUGET_KEY}} --skip-duplicate
\ No newline at end of file
+        run: |
```
[the remark this answers is not in the document]
Comment (MartinCarpentier) [reply .github/workflows/release.yml:51]: You are completely right. We do have the version number, and in theory the wildcard should not be needed.

Arh and i see, i just read around and it seemed like PowerShell didn't have as good wildcard support as bash, and the error was this.

image

So it seemed like the wildcard was taken as the path itself.

Should i fix the wildcard, or should i just bind to the actual nuget package directly.
What's your thought?

[the remark this answers is not in the document]
Comment (matt-lethargic) [reply .github/workflows/release.yml:51]: Personally I'd use the exact path using the version number if possible. But it's entirely up to you. If it works, it works.

[the remark this answers is not in the document]
Comment (MartinCarpentier) [reply .github/workflows/release.yml:51]: Yes, that seems like the best solution.
I will fix it.

**Pull request merged**